VideoApp/VideoApp/Stores/AppSettings/TwilioEnvironment.swift

```diff
@@ -16,7 +16,7 @@
 
 import Foundation
 
-enum Environment: String, SettingOptions {
+enum TwilioEnvironment: String, SettingOptions {
```

Comment:
This was conflicting with a SwiftUI symbol and probably better to be more specific anyway.